suite_test.go

```diff
@@ -1,8 +1,9 @@
 package yaml_test
 
 import (
-	. "gopkg.in/check.v1"
```

Comment:
I this necessary? 

If not, could we keep the number of lines changed to a minimum?  Don't want to differ _too_ far from upstream in case we need to pull in changes.